apis/kueue/v1beta1/fairsharing_types.go

```diff
@@ -35,6 +35,7 @@ type FairSharing struct {
 	Weight *resource.Quantity `json:"weight,omitempty"`
 }
 
+// fairSharing contains the information about the current status of fair sharing.
```

Comment:
I'm not sure about the naming here - I understand why you did this but maybe it should be consistent with the type.

wdyt @mimowo @tenzen-y

Reply:
looking at other comments we should probably align with the type name indeed: https://github.com/kubernetes-sigs/kueue/pull/4561/files#diff-fa7799c26041d219ab624bc2069198280537236008ff41d199593618b900dcb9L255.

Also, this is named like this in the core k8s: https://github.com/kubernetes/kubernetes/blob/af3b4cd57a5e3a1dbdf9115b7ac71fcc309c7722/pkg/apis/core/types.go#L2956-L2957, even though the type is referenecd by "Conditions" field. You never know where someone references from your type.

Reply:
Not sure about it. I think we need to set with lower case due to on generation we will see FairSharingStatus.

https://github.com/kubernetes-sigs/kueue/blob/f56055462247e124b5de5d9e7104d826b0ed7069/charts/kueue/templates/crd/kueue.x-k8s.io_cohorts.yaml#L244-L248

Reply:
Yeah, @mbobrovskyi is right. The generated documentation is the reason. API documentation is referred by users and they use YAML instead of code.